Just entered into 0x05-python-exceptions in the ALX project
